src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -399,7 +399,8 @@ public void patronNoticeOnCheckInIsSentWhenCheckInLoanNoticeIsDefinedAndLoanExis
     useLoanPolicyAsFallback(
       loanPoliciesFixture.canCirculateRolling().getId(),
       requestPoliciesFixture.allowAllRequestPolicy().getId(),
-      noticePoliciesFixture.create(noticePolicy).getId());
+      noticePoliciesFixture.create(noticePolicy).getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId());
```

Comment:
This code exists 5 times. Can you extract it into a method? Example:

    private void use(NoticePolicyBuilder noticePolicy) {
      useLoanPolicyAsFallback(
      loanPoliciesFixture.canCirculateRolling().getId(),
      requestPoliciesFixture.allowAllRequestPolicy().getId(),
      noticePoliciesFixture.create(noticePolicy).getId(),
      overdueFinePoliciesFixture.facultyStandard().getId());
    }

Reply:
Done.